m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -242,18 +242,18 @@ io_combinations()
 	# unit size in K
 	for unit_size in 4 8 32 128 512 2048 4096
 	do
-	    stripe_size=$(($unit_size '*' $data_units))
+	    stripe_size=$(($unit_size '*' "$data_units"))
```

Comment:
The same problem.

Reply:
When I tried this, I understood that this is a mathematical operation in which we should not give quotes hence removing it.
Specially we should not quote '*'. Hence will leave it untouched.